### PR TITLE
0.1.0

### PR DESCRIPTION
**Pull request by sergei-bondarenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,195 @@
...
+
+- Prepare server (add elastic repo)
+- [Optional] Create folder(s) for custom paths
+- Install metricbeat
+- Copy configuration file
+
+Requirements
+------------
+
+- Minimal Version of the ansible for installation: 2.5
```
Comment (tgadiev) [inline README.md]: I would say 2.7 or 2.8

File: README.md [hunk 2]
```diff
@@ -1,2 +1,195 @@
...
+ - **Supported OS**:
+   - CentOS
+     - 7, 8
+   - Ubuntu
+     - 16.04
```
Comment (tgadiev) [inline README.md:38]: Ubuntu 18?

File: defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+mb_conf_dir: '{{ default["mb_conf_dir"] }}'
+mb_data_dir: '{{ default["mb_data_dir"] }}'
+
+mb_ssl_key: '{{ default_ssl["key"] }}'
+mb_ssl_certificate: '{{ default_ssl["certificate"] }}'
+mb_ssl_certificate_authorities: '{{ default_ssl["certificate_authorities"] }}'
+
+win_download_path: '{{ ansible_env.TEMP }}/metricbeat'
+
+input_logpath: '{{ default_logpath }}'
```
Comment (tgadiev) [inline defaults/main.yml]: What is this variable for?

File: meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+dependencies: []
+
+galaxy_info:
+  role_name: metricbeat
+  author: Lean Delivery Team <team@lean-delivery.com>
+  description: Role for Metricbeat installation
+  company: EPAM Systems
+  issue_tracker_url: https://github.com/lean-delivery/ansible-role-metricbeat/issues
+  license: Apache
+  min_ansible_version: 2.8
```
Comment (tgadiev) [inline meta/main.yml]: 2.7 may be?

File: meta/main.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+  platforms:
+    - name: EL
+      versions:
+        - 7
+        - 8
+    - name: Ubuntu
+      versions:
+        - xenial
```
Comment (tgadiev) [inline meta/main.yml:20]: bionic?

File: meta/main.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        - stretch
+    - name: "Amazon"
+      versions:
+        - "all"
+
+  galaxy_tags:
+    - metricbeat
+    - elasticsearch
+    - elk
+    - logging
```
Comment (tgadiev) [inline meta/main.yml]: ???

File: molecule/resources/tests/verify.yml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
+  hosts: all
+  tasks:
+  - name: Example assertion
+    assert:
+      that: true
```
Comment (tgadiev) [inline molecule/resources/tests/verify.yml]: Please, add valid verification steps

File: tasks/Debian.yml [hunk 8]
```diff
@@ -0,0 +1,40 @@
+---
+- name: Set Metricbeat version if defined
+  set_fact:
+    mb_package_name: '{{ mb_package_name }}={{ metricbeat_version }}'
+  when: metricbeat_version.find('.x') == -1
+
+- block:
+    - name: Install dependency for Debian
+      package:
```
Comment (tgadiev) [inline tasks/Debian.yml]:
```suggestion
      apt:
```

File: tasks/Win32NT.yml [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+  win_file:
+    path: '{{ path_folder.value }}'
+    state: directory
+  with_dict: '{{ path }}'
+  loop_control:
+    loop_var: path_folder
+
+- name: Download metricbeat
+  win_get_url:
+    url: 'https://artifacts.elastic.co/downloads/beats/metricbeat/metricbeat-{{ metricbeat_version }}-windows-x86_64.zip'
```
Comment (tgadiev) [inline tasks/Win32NT.yml]: I would move full url to vars

File: vars/Win32NT.yml [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+  mb_log_dir: C:\program files\metricbeat\logs
+  mb_conf_dir: C:\program files\metricbeat
+  mb_data_dir: C:\program files\metricbeat\data
+
+default_ssl:
+  key: c:\tls\private\server.key
+  certificate: c:\tls\certs\server.pem
+  certificate_authorities: c:\CA\ca-root.pem
+
+default_logpath: c:\windows\*.log
```
Comment (tgadiev) [inline vars/Win32NT.yml]: Why is this for?

**Review by tgadiev — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sergei-bondarenko) [reply README.md:38]: Ubuntu 18 excluded from tests, see https://github.com/lean-delivery/ansible-role-metricbeat/pull/1/commits/6c8c14d2d2e559353ab54412b2b0906449af92c1.

File: meta/main.yml (hunk 5 above)
Comment (sergei-bondarenko) [reply meta/main.yml:20]: Ubuntu 18 excluded from tests, see https://github.com/lean-delivery/ansible-role-metricbeat/pull/1/commits/6c8c14d2d2e559353ab54412b2b0906449af92c1.

**Review by tgadiev — APPROVED** [review]

**Pull request merged**